pegasus/sites.v3/code.org/public/hourofcode/overview.haml

```diff
@@ -1,16 +1,12 @@
 ---
 title: Hour of Code Teacher Resources
 theme: responsive
-social:
```

Comment:
Why remove these?

Reply:
The opengraph tags for this page are being implemented on `social_metadata.rb` so I cleaned up the old tags. https://github.com/code-dot-org/code-dot-org/blob/193e93297dc30ce23772b2cf50b1fdccfe833db3/pegasus/src/social_metadata.rb#L144-L148